src/main/java/fly/wild/wizards/serverside/ServerConnector.java

```diff
@@ -0,0 +1,328 @@
+import java.util.UUID;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.dmr.ModelNode;
+import org.jboss.as.cli.ControllerAddress;
+import org.jboss.as.cli.Util;
+import org.jboss.as.cli.impl.ModelControllerClientFactory;
+import org.jboss.as.cli.operation.OperationFormatException;
+import org.jboss.as.cli.operation.impl.DefaultOperationRequestBuilder;
+
+import fly.wild.wizards.tlswizard.controller.OneWayTLSConfigurationConfiguration;
+import fly.wild.wizards.tlswizard.controller.TLSConfiguration;
+
+public class ServerConnector {
+
+	private String ipAddress;	
+	private String distinguishedName;
+	private String alias;
+	private StringBuilder logMessage;
+	private final String STRING_CLEAR_TEXT_VALUE_KEY_STORE = "keystorepass";
+	private final String STRING_ALGORITHM_VALUE = "RSA";
+	private final int INT_KEY_SIZE_VALUE = 2048;
+	private final int INT_VALIDITY_VALUE = 365;
+	private final String STRING_SERVER_VALUE = "default-server";
+	private final String STRING_HTTPS_LISTENER_VALUE = "https";
+	private String genKeyStoreName;
+	private String genKeyManagerName;
+	private String genServerSSLContext;
+	private String padding;
+	private String keyStoreFileName;	
+	private ControllerAddress connectionAddress;
+	private OneWayTLSConfigurationConfiguration oneWayTLSConfiguration;
+	private TLSConfiguration tlsConfiguration;
+	private final int connectionTimeOut = 5000;
+	
+	public ServerConnector(TLSConfiguration tlsConfiguration, OneWayTLSConfigurationConfiguration oneWayTLSConfiguration) {
+		LocalTime localTime = LocalTime.now();
+		UUID uuid=UUID.randomUUID();
+		
+		this.oneWayTLSConfiguration = oneWayTLSConfiguration;
+		this.tlsConfiguration = tlsConfiguration;
+		this.ipAddress = this.tlsConfiguration.getServerIP();
+		
+		this.padding = uuid.toString();
+		
+		this.genKeyStoreName = "wizgenks" + this.padding;
+		this.genKeyManagerName = "wizgenkm" + this.padding;
+		this.genServerSSLContext = "wizgenssl" + this.padding;
+		this.keyStoreFileName = this.oneWayTLSConfiguration.getKeyStoreFileNameValue();
+		this.alias = this.oneWayTLSConfiguration.getFirstAndLastNameValue();		
+		this.distinguishedName = "CN=" + this.oneWayTLSConfiguration.getFirstAndLastNameValue() +
+				";" + "OU=" + this.oneWayTLSConfiguration.getOrganizationalUnitValue() +
+				";" + "O=" + this.oneWayTLSConfiguration.getOrganizationValue() +
+				";" + "L=" + this.oneWayTLSConfiguration.getCityOrLocalityValue() +
+				";" + "ST=" + this.oneWayTLSConfiguration.getStateOrProvinceValue() +
+				";" + "C=" + this.oneWayTLSConfiguration.getCountryCodeValue();
+
+		this.logMessage = new StringBuilder("This is the TLSWizard run of "+localTime.toString()+"\n");
+		connectionAddress = new ControllerAddress("remote+http",ipAddress,9990);	
+		
+	}
+	
+	
+	public boolean configureOneWayTLS() {
+		
+		boolean success = true;
+		
+		// Create a key store		
+		DefaultOperationRequestBuilder keyStoreBuilder = this.buildKeyStoreNode();
```

Comment:
Just a minor, I think I would rename these methods slightly to make it clearer what they do. Just some ideas:

buildKeyStoreNode -> createKeyStoreAddOperationRequestBuilder

buildStoreCertificateBuilder -> createGenerateCertificateOperationRequestBuilder

I replaced the `build` with `create` as the buildRequest is not actually called in the methods. But if you decide to place the buildRequest() calls inside of the methods you can rename to `buildKeyStoreAddOperation` and `buildGenerateCertificateOperation` for example

Reply:
Another thought is that we can remove the variables here and create these operations in the client.execute() methods:

client.execute(buildKeyStoreAddOperation())